app/views/shared/_user_util_links.html.erb

```diff
@@ -38,19 +42,19 @@
         </li>
       </ul>
     </li>
-  <% ### BEGIN CUSTOMIZATION elr - remove login link from header (see footer for new location) %>
-  <% # else % >
-     # <li class="nav-item">
-     #   < %= link_to t('spotlight.header_links.login'), main_app.new_user_session_path, class: 'nav-link' % >
-     # </li>
-  %>
-  <% ### END CUSTOMIZATION %>
+    <% ### BEGIN CUSTOMIZATION elr - remove login link from header (see footer for new location) %>
```

Comment:
This section is commented out - can it be removed?

Reply:
I'm guessing that this was commented out previously to keep track of what was removed as the customization.

Reply:
aha! so it is!